softhdmenu.h

```diff
@@ -23,98 +22,44 @@
 #define __SOFTHDMENU_H
 
 #include "softhddevice.h"
-#include "softhdmenu.h"
-
-class cSoftHdDevice;
 
 /*****************************************************************************
- * cMenuSetupSoft - Setup menu
+ * cSoftHdMenu
  ****************************************************************************/
 
-/**
- * cMenuSetupSoft - SoftHdDevice plugin menu setup page class
- */
-class cMenuSetupSoft:public cMenuSetupPage
+typedef enum {
+	hksInitial,
+	hksBlue
+} HkState;
```

Comment:
Abbreviations in names can get out of hand easily, if excessively used. Why not HotkeyState? You could also use `HotkeyState::Initial` when using it. Then, the hks prefix in the name could be omitted.